pyvrp/cpp/Solution.cpp

```diff
@@ -108,7 +109,7 @@ bool Solution::operator==(Solution const &other) const
 }
 
 Solution::Solution(ProblemData const &data, RandomNumberGenerator &rng)
-    : neighbours(data.numClients() + 1, {0, 0})
+    : neighbours(data.numClients() + 1)
```

Comment:
```suggestion
    : neighbours(data.numClients() + 1, std::nullopt)
```
is a bit more explicit